### DISCUSSION TITLE
Remove artificial limitations

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16649] [General]
As a user, I do not want artificial limitations imposed on me for no apparent reason.

This includes:
* The maximum of 3 pinned messages
* The maximum of 2000 characters per message

Matrix allows for 1024 pinned messages, and 65535 characters per message. These aren't artificial, but rather technical limitafions.

Please consider removing the arbitrary artificial limitations.

**Comment by flexsurfer** [comment]
cc @cammellos
